api/src/main/java/net/kyori/adventure/sound/Sound.java

```diff
@@ -144,6 +144,14 @@
    */
   float pitch();
 
+  /**
+   * Gets the {@link SoundStop} that will stop this specific sound.
+   *
+   * @return the sound stop
+   * @since 4.8.0
+   */
+  @NonNull SoundStop stopper();
```

Comment:
Unsure of this name - @zml2008?

Reply:
hm, maybe `asStop`?

Reply:
Yeah I was a bit conflicted about this, I chose stopper as it returns the thing that stops the sound and is quite clear that it wouldn't actually stop the sound (as opposed to, say, `stop`).

I like `asStop` although I don't think it's as clear as `stopper`, not that it matters too much. Happy to change if needs be!

Reply:
@kashike i don't have a super strong opinion either way at the moment, would you decide (or flip a coin?)

Reply:
`asStop` is actually more in keeping with the rest of the sound API (`asSource` for example).

Reply:
alright then, let's go with `asStop`